_posts/en/newsletters/2020-11-18-newsletter.md

```diff
@@ -0,0 +1,96 @@
+---
+title: 'Bitcoin Optech Newsletter #124'
+permalink: /en/newsletters/2020/11/18/
+name: 2020-11-18-newsletter
+slug: 2020-11-18-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter contains a warning about backdoored VM images.
+Also included are our regular sections with updates-about various projects.
```

Comment:
omit hyphen in "updates-about"

Reply:
@bitschmidty how much do you want this?  I've been trying to switch to a more generic formulation that doesn't need to get updated when we drop sections at the last moment (as often happens with the releases/RCs).  OTOH, if you want the call outs to the particular sections each week, I'm happy to do that.

Reply:
@harding Im not strong on it exactly as is, was just pulling something previous for consistency. I do think the weekly sections (review club, client/services) are worth a call out due to their consistent schedule.